src/csharp/Pester/Tracing/CodeCoverageTracer.cs

```diff
@@ -53,14 +85,21 @@ public void Trace(IScriptExtent extent, ScriptBlock _, int __)
                 return;
 
 
-            var point = lineColumn[key2];
-            if (point.Hit == true)
+            var points = lineColumn[key2];
+            if (points.TrueForAll(a => a.Hit))
```

Comment:
This seems overkill, we are not adding hits once the collection is established, so we might just check the first 1 if it is true or false.